packages/Takeover/src/components/Content.js

```diff
@@ -4,9 +4,8 @@ import tokens from "@paprika/tokens/lib/tokens";
 
 const Content = styled.div`
   background-color: ${tokens.color.white};
-  border: 1px solid ${tokens.border.color};
   border-radius: ${tokens.border.radius};
```

Comment:
@DanilAgafonov This border radius should be `border-radius: ${tokens.card.borderRadius}`

Reply:
Hmmm, looks like we are trying to create card/paper component.

@johnpeele what do you think, if I create `@paprika/paper` component? Something like https://material-ui.com/components/paper/